apps/user-profile/src/styles.scss

```diff
@@ -118,3 +118,8 @@ th, td {
 .app-min-width {
   min-width: 400px;
 }
+
+.mat-tooltip {
+  font-size: 14px !important;
+  word-wrap: break-word !important;
+}
```

Comment:
Be careful with this. This way, you will affect all of the tooltips in the whole application. I don't know if the font-size change is necessary. About the word-wrap, I would maybe try to limit this change only for a certain tooltip.

Reply:
We have the same "global" class in admin-gui, so I used it also in user-profile.